plotly/offline/offline.py

```diff
@@ -210,12 +211,22 @@ def iplot(figure_or_data, show_link=True, link_text='Export to plot.ly',
                                has become outdated with your version of
                                graph_reference.json or if you need to include
                                extra, unnecessary keys in your figure.
+    image (default=False) -- If True, the image of the current plot
+        will be downloaded.
+    format (default='png') -- Specifies the format of the image to be
+        downloaded if `downlowad_image` is True.
```

Comment:
🐄 Can we just do `image=None|'png'|...`? That feels just an understandable to me, but it only requires a single kwarg. Also, it makes it such that `format` doesn't do *nothing* if `image=False`.